cmake/configs/default.cmake

```diff
@@ -176,7 +176,7 @@ hunter_default_version(autobahn-cpp VERSION 0.2.0)
 hunter_default_version(autoutils VERSION 0.3.0)
 hunter_default_version(aws-c-common VERSION 0.2.1-p0)
 hunter_default_version(benchmark VERSION 1.4.0)
-hunter_default_version(binaryen VERSION 1.38.28-p1) 
```

Comment:
Please send the changes that are not related to `ZLIB` as a separate pull request.

Reply:
Sorry, I will turn off my whitespace cleaner.

Reply:
> Sorry, I will turn off my whitespace cleaner.

No, it's fine, just send it separately.